setup.py

```diff
@@ -1,12 +1,23 @@
 from setuptools import setup
 
-__version__ = "1.0"
+__version__ = "1.1"
```

Comment:
Just checking - when we want to make a release, we bump this version, then run the new packaging instructions in the readme?

Reply:
That would do for now. But I'm about to trial a github action which will do this for us. Something like "we pushed a tag starting with v" was my thinking. Or it could just be a button press.

Reply:
Auto-releasing to PyPI on an action would probably be a good step. I think it would probably be useful to port it to our other modules & module templates too, that would probably take a lot of the burden of releasing off.

Reply:
You have cunningly managed to uncover my ulterior motive here